controllers/case_studies/getAll/get.js

```diff
@@ -0,0 +1,11 @@
+
+const CaseStudy = require('../../../models/caseStudy/casestudy');
+
+module.exports = (req, res) => {
+  CaseStudy.getAll((err, caseStudy) => {
```

Comment:
Burasi genel bi tuhaf. Bi kere error formatını XMLHTTP Request'ler için olan genel hale getir. Bir de getAll kötü bir fonksiyon ismi, getAllCaseStudies falan yap bence